numba/cuda/cudadrv/libs.py

```diff
@@ -80,67 +79,97 @@ def _get_source_variable(lib, static=False):
         return get_cuda_paths()[dir_type].by
 
 
-def test(_platform=None, print_paths=True):
+def test():
     """Test library lookup.  Path info is printed to stdout.
     """
     failed = False
 
     # Check for the driver
     try:
         dlloader, candidates = locate_driver_and_loader()
-        locations = ", ".join(candidates)
-        print(f'Finding driver from candidates: {locations}...')
+        print('Finding driver from candidates:')
+        for location in candidates:
+            print(f'\t{location}')
         print(f'Using loader {dlloader}')
-        print('\ttrying to load driver', end='...')
+        print('\tTrying to load driver', end='...')
         dll, path = load_driver(dlloader, candidates)
-        print(f'\tok, loaded from {path}')
+        print('\tok')
+        print(f'\t\tLoaded from {path}')
     except CudaSupportError as e:
         print(f'\tERROR: failed to open driver: {e}')
         failed = True
 
+    # Find the absolute location of the driver on Linux. Various driver-related
```

Comment:
The absolute path may contain information that users might not want to share, however the point of this diagnostic is that the absolute path is reported. The `numba -s` tool has a large printed warning at the end of its output with regards to checking and redacting the output before sharing which I imagine covers this.